lib/region/EU/countryCodes.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file List of country codes legally considered part of the EU region.
+ */
+
+var countryCodes = {
+  AT: true, // Austria
+  BE: true, // Belgium
+  BG: true, // Bulgaria
+  CH: true, // Switzerland
+  CZ: true, // Czech Republic
+  DE: true, // Germany
+  DK: true, // Denmark
+  ES: true, // Spain
+  EE: true, // Estonia
+  FI: true, // Finland
+  FR: true, // France
+  GB: true, // Great Britain / UK
+  GR: true, // Greece
+  HR: true, // Croatia
+  HU: true, // Hungary
+  IE: true, // Ireland
+  IS: true, // Iceland
+  IT: true, // Italy
+  LT: true, // Lithuania
+  LU: true, // Luxembourg
+  LV: true, // Latvia
+  NL: true, // Netherlands
+  NO: true, // Norway
+  PL: true, // Poland
+  PT: true, // Portugal
+  RO: true, // Romania
+  SE: true, // Sweden
+  SI: true, // Slovenia
+  SK: true  // Slovakia
+};
```

Comment:
I believe the performance difference you were seeing was due to using Boolean coercion in one test and not the other. I edited the tests a bit to show both styles with and without boolean lookup. Map lookup is significantly faster than looking it up in an array. This is expected, as a lookup in a hash map is O(1), whereas lookup in a map is O(n).
http://jsperf.com/jst-7/2